server/ServerFramework.cpp

```diff
@@ -183,8 +183,8 @@ auto PythonServer::FillListPlayers(std::vector<PlayerSetupData>& players) const
     const py::extract<py::list> py_players(r);
     if (py_players.check()) {
         py::stl_input_iterator<PlayerSetupData> players_begin(py_players), players_end;
-        players.reserve(std::distance(players_begin, players_end));
```

Comment:
regardless of the rest, why remove the `reserve`?

Reply:
I'll try it.

Reply:
Yes, this is consuming input so I used `len` function to reserve vector capacity.